### DISCUSSION TITLE
No in-cockpit sounds

### DISCUSSION THREAD

**Discussion by comanche001** [flybywiresim/aircraft#4551] [Help] [answered]
After the update to 0.6 I don't have in-cockpit sounds anymore. (experimental and developer version)
I do hear environment sounds and also outside everything is fine.
So when I turn on battery and external power -> no sounds.
Other planes: no problem. All sounds are working.

Is there a configuration somewhere that I missed?

**Comment by hotshotp** [comment, marked as answer]
You have to delete a32nx from community and reinstall the mod now that it's standalone. For more info look at the sticky message in the support channel in discord

**Reply by comanche001** [reply]
That worked! Thank you!
